apps/docs/content/components/button.mdx

```diff
@@ -11,14 +11,13 @@ links:
 <PreviewComponent src="buttons/docs/preview"/>
 
 ## Usage
-```shell
+```tsx
 import { Button } from "@hopper-ui/components";
 ```
 ```tsx showLineNumbers
 <Button>save</Button>
 ```
 ## Props
-The table below contains all types of the props available in title component.
 
 <PropTable component="Button" />
```

Comment:
Comment before the review, just visual QA: There is 2 props that look out of place. 

`autoFocus` and `type` are in the Accessibility section, but aren't related to accessibility. i would expect those 2 properties to be in the main section